Common/Tests/MockVsTests/MockVsTests.csproj

```diff
@@ -86,15 +77,59 @@
     <Reference Include="WindowsBase" />
   </ItemGroup>
   <Choose>
-    <When Condition="$(VSTarget) == '15.0'">
+    <When Condition="$(VSTarget) == '10.0'">
       <ItemGroup>
-        <Reference Include="Microsoft.VisualStudio.Shell.Framework, Version=$(VSTarget).0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" />
+        <Reference Include="Microsoft.VisualStudio.Shell.10.0" />
+        <Reference Include="Microsoft.VisualStudio.Shell.Immutable.10.0, Version=10.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL" />
       </ItemGroup>
     </When>
-    <Otherwise>
+    <When Condition="'$(VSTarget)' == '11.0'">
```

Comment:
Do we really need `'$(VSTarget)' == '11.0'`?

Reply:
We've never officially dropped old versions from the shared code ("Common" folder). So until we do that, yes. (And [PHP Tools](https://www.devsense.com/en/download) is still supporting VS 2010 using this code, so we should run it past them before making any significant change here.)